addons/popochiu/engine/objects/gui/components/sound_volumes/sound_volumes.gd

```diff
@@ -63,9 +62,10 @@ func _on_audio_manager_ready() -> void:
 		$SlidersContainer.add_child(slider)
 		
 		slider.value_changed.connect((
-			func (value: float, bus_name: String):
-				E.am.set_bus_volume_db(
-					bus_name,
+			func (value: float, bus_name_param: String):
+				@warning_ignore("standalone_ternary")
```

Comment:
Darn, I'm torn. Passing a value as a parameter like this is very punk and very smart at the same time...

Reply:
I'm going to say that I have no idea what this piece of code is doing. Is it possible to simplify this by any chance?

Reply:
I tend to agree with @balloonpopper on that. It's nothing bad, and it's something pretty readable to (ex.) a skilled javascript developer. I can follow along (despite not being a skilled js dev myself), bit the smell here is the need to suppress a warning. It's not wrong, it's stretching the language beyond its paradigm.

Reply:
Thanks. I put the logic in a separate function now.